RadarSDK/RadarLocationManager.m

```diff
@@ -519,7 +520,7 @@ - (void)replaceSyncedGeofences:(NSArray<RadarGeofence *> *)geofences {
                     NSString *notificationRepeats = [geofence.metadata objectForKey:@"radar:notificationRepeats"];
                     if (notificationRepeats) {
                         repeats = [notificationRepeats boolValue];
-                        [[RadarLogger sharedInstance] logWithLevel:RadarLogLevelDebug message:[NSString stringWithFormat:@"repeats is bool vaue: %d", repeats]];
+                        [[RadarLogger sharedInstance] logWithLevel:RadarLogLevelDebug message:[NSString stringWithFormat:@"Notification repeats | repeats = %d", repeats]];
```

Comment:
This copy will be confusing in debug logs, as it'll appear as "Notification repeats" in the main log line, only showing "repeats = false" once you click in.